### PR TITLE
fix[bug] issue compatibility with LP new version 4.2.2.3

### PR DESCRIPTION
**Pull request by asdzakky**

### CODE REVIEW

File: functions/user.php [hunk 1]
```diff
@@ -14,20 +14,50 @@ function sejolilp_get_user_purchased_courses( $user_id = 0 ) {
     $user    = learn_press_get_user( $user_id );
     $query   = $profile->query_courses( 'purchased' );
 
-    if( $query['items'] ) :
+    if( $learnpress_version <= '4.2.2.2' ) :
```
Comment (orangerdev) [inline functions/user.php]: [pakai fungsi php version_compare](https://www.php.net/manual/en/function.version-compare.php)

**Review by orangerdev — CHANGES_REQUESTED** [review]

File: functions/user.php (hunk 1 above)
Comment (asdzakky) [reply functions/user.php]: ok done pak @orangerdev

Comment (orangerdev) [reply functions/user.php]: eh ini kok belum berubah ya mas?

**Review by orangerdev — CHANGES_REQUESTED** [review]

File: functions/user.php (hunk 1 above)
Comment (asdzakky) [reply functions/user.php]: lah iiya kok gk berubah ya, udh sy push pdhal, cba sy cek lg pak

**Review by orangerdev — APPROVED** [review]

**Pull request merged**